### PR TITLE
[hi] Add Hindi glossary entry for "daemonset"

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
Localizes content/en/docs/reference/glossary/daemonset.md to Hindi content/hi/docs/reference/glossary/daemonset.md

### Description
#Fixes Localizes content/en/docs/reference/glossary/daemonset.md to Hindi content/hi/docs/reference/glossary/daemonset.md

### Issue

Closes: #

### CODE REVIEW

File: content/hi/docs/reference/glossary/daemonset.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: डेमनसेट (DaemonSet)
```
Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/daemonset.md:2]: DaemonSet in aka

File: content/hi/docs/reference/glossary/daemonset.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+short_description: >
+  एक पॉड की एक कॉपी क्लस्टर के विभिन्न नोड्स पर चल रही है, यह सुनिश्चित करता है।
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक  {{< glossary_tooltip text="पॉड" term_id="pod" >}} की एक कॉपी  {{< glossary_tooltip text="क्लस्टर" term_id="cluster" >}} के विभिन्न नोड्स पर चल रही है, यह सुनिश्चित करता है।
```
Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/daemonset.md:15]:
```suggestion
यह सुनिश्चित करता है कि {{<glossary_tooltip text="पॉड" term_id="pod">}} की एक प्रति {{<glossary_tooltip text="क्लस्टर" term_id="cluster" >}} में नोड्स के एक सेट पर चल रही है।
```

File: content/hi/docs/reference/glossary/daemonset.md (hunk 1 above)
Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/daemonset.md:2]: fundamental ->  core-object ->  workload ? Where exactly?

**Comment by CloudShiva7cr** [comment]
@niranjandarshann @utkarsh-singh1 Can you review once, have made necessary changes as per your suggestions,  Thanks!!

**Comment by CloudShiva7cr** [comment]
@divya-mohan0209 Can you review once and let me know if any more changes needed!! Thanks

**Pull request closed without merging**